packages/components/inputs/select-utils/src/export-types.ts

```diff
@@ -1,10 +1,7 @@
 export type { TClearIndicatorProps } from './clear-indicator/clear-indicator';
-export type {
-  TTagRemoveSelectProps,
```

Comment:
Not sure if this constitutes a breaking change?

Reply:
That's a good point.

This would actually be a breaking change, although this package is more internal than external.

What we can do is to keep those types just for backwards compatibility and mark them as deprecated. We can remove them later when we plan for a major `ui-kit` release.

Since this types are not needed anymore in their components, I would maybe add something like this to the `types.ts` file:

```ts

/**
 * @deprecated Use `TTagRemoveProps['selectProps]` instead
 */
export type TTagRemoveSelectProps = {
  isReadOnly: boolean;
} & MultiValueGenericProps['selectProps'];

/**
 * @deprecated Use `TWrapperWithIconProps['selectProps]` instead
 */
export type TWrapperWithIconSelectProps = {
  iconLeft?: ReactElement;
};
```

Reply:
I included these types here: cb411c1ce2bccbe8808d466e8f87a738c3ab918c